scripts/install

```diff
@@ -176,28 +221,62 @@ ynh_script_progression --message="Securing files and directories..."
 
 # Set permissions to app files
 chown -R $app: $final_path
+chmod -R 775 "$final_path/storage"
```

Comment:
```suggestion
chmod 775 "$final_path/storage"
```

(Probably doesn't matter so much here, because dir is probably empty, but c.f. other comment)